### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by tlkiong** [james2m/seedbank#87]

**Comment by Eusebius1920** [comment]
**Still**: as of now the project is working fine, even with the latest rails versions or are you experiencing any issues?

**Comment by tlkiong** [comment]
@Eusebius1920 It isn't about experiencing any issues or not. I was just wondering if this is still active and / or maintained.

**Comment by jcquarto** [comment]
I agree, even a minor commit that adds "works with Rails 6 and Rails 7" in the README would have been a comfort to see. My first reaction was "oh, maybe I should look for a more recent other gem" or somesuch.

**Comment by seuros** [comment]
It works perfectly.

**Comment by hockleyd** [comment]
There are three pull requests, that have been merged and/or have no conflicts, and no comments.  So there is a valid question of whether the project is being actively maintained?  I encountered a bug as far as camel-cased / underscore table names in the dependency, but I'm wondering if it's worth chasing a bug or a PR if there's no activity?

**Comment by tlkiong** [comment]
i just forked this to work
